### DISCUSSION TITLE
关于onedrive business 的webdav支持

### DISCUSSION THREAD

**Discussion by tmdjiuji** [AlistGo/alist#300] [Ideas]
部分onedrive business是不支持api的，请问是否可以支持下onedrive business 的webdav(包含世纪互联）？

**Comment by xhofe** [comment]
没有API怎么支持

**Reply by tmdjiuji** [reply]
以下链接是rclone那里关于sharepoint/onedrive business的webdav介绍：https://rclone.org/webdav/ ，希望如果可以就支持兼容下

**Reply by xhofe** [reply]
onedrive的WebDAV并不是通用的WebDAV协议，所以暂无法支持。
